tests/wasmbus.rs

```diff
@@ -390,24 +390,21 @@ expected: {expected_labels:?}"#
         .context("failed to parse `rust-httpserver` provider key")?;
     let httpserver_provider_url = Url::from_file_path(test_providers::RUST_HTTPSERVER)
         .expect("failed to construct provider ref");
-    let httpserver_provider_link_name = "link-name-1";
```

Comment:
Why switch to magic strings?

Reply:
For clarity, simplifying debugging and being explicit.
Note, that in doing so I've fixed an invalid link name specified in the link deletion.

Reply:
Not going to block on this since it's a test, but my $0.02: copy-paste errors are copy-paste errors, regardless of whether they're a named var or a string. The advantage of a named var over a string is to prevent fat-finger typos, like passing `htpserver` as a link name when it should be `httpserver`. If it's a named variable, typos won't compile

Reply:
Sure, the advantage of plain strings is maintenance, this string has to be duplicated in at least 5 different crates and indirection here just makes it more difficult to keep track of these. Looking at a single call in isolation should be enough to figure out what link name is used

Reply:
Did you mean to respond to the other thread? I'm not sure I follow the reference to multiple crates, since this is all in one test file

Reply:
See the updated diff, names must match in 3 Wasm test binaries and in 2 tests - one you commented upon and a `wasmcloud_runtime`-level test